aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,29 +170,38 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are four types of capitalization. 
+1. Sentence case, where the first letter is a capital, the second and all subsequent words begin with lowercase letters.
```

Comment:
all subsequent words are not always lowercase in a sentence-case sentence.  Proper nouns should always be capitalized.  BTW, if we don't already, we should talk about not inventing new proper nouns.  For example, folks tend to want to turn any tech term into a proper noun: Send an Email regarding the Power Button in order to Reset the System. (none of these are proper nouns)

Reply:
Re: "subsequent" you're right! I just had to rewrite this from memory quickly because I thought my branch was lost and I want to be able to have this published by Tuesday for the presentation to GLC. I will re-word :) TY!

Also, I forwarded you something that Edgar had written. It's a table that also has a column for how big of an impact it would make on their systems. If your schedule allows @ericsoderberghp could we talk about this soon?

Reply:
The text that's already published states "Fight the urge to elevate normal nouns to proper noun status..." ;)